data/standings.py

```diff
@@ -0,0 +1,100 @@
+import requests
+import re
+
+class Standings:
+    __URL = 'https://statsapi.mlb.com/api/v1/standings?season={year}&leagueId=103,104&date={month:0>2}/{day:0>2}/{year}&division=all'
```

Comment:
Can you extract the `leagueId`'s out to constants? Magic numbers aren't very fun

Reply:
@pawptart I'm moreso trying to understand what 103/104 actually are. Splitting them out of the main URL string is great but I'm just looking for some human readable context

Reply:
Is it NL/AL?

Reply:
My bad, I must have been distracted yesterday when I put that up.

Yes, it's just NL/AL.

Reply:
What we'd rather see is something like `LeagueID_NL = 103` and `LeagueID_AL = 104` ... or whichever ID actually means which league. The fact that we don't know is the best reason to break them out into descriptive constants.

Reply:
Totally understand! Should be fixed now.